presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -82,6 +83,18 @@ public boolean schemaExists(ConnectorSession session, String schemaName)
         return ImmutableList.copyOf(jdbcClient.getSchemaNames(JdbcIdentity.from(session)));
     }
 
+    @Override
+    public Optional<PrestoPrincipal> getSchemaOwner(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        return Optional.empty();
```

Comment:
Is this correct? I guess RDBMS might have a notion of schema owner. Maybe we should leave it unimplemented?

Reply:
Throw UnsupportedException?
What I did was to have an implementation that does not fail for every connector that supports `CREATE SCHEMA`. Wouldn't it be weird to have `SHOW CREATE SCHEMA` fail with an exception here?
No problem removing the implementation here, we have to just be aware of the implications.

Reply:
I think we might add properties for schemas in the Jdbc Connector in the future. So getSchema  Owner (IMHO) should not fail.

Reply:
> Wouldn't it be weird to have SHOW CREATE SCHEMA fail with an exception here?

If it is better to fail than return incorrect data. In other words, JDBC connectors is a pandora box let's not open it for now.

Reply:
Ok... Lemme do that.

Reply:
Also... I assume the same holds for KuduMetadata, BlackHoleMetaData, MemoryMetadata, PhoenixMetadata?

The logic I followed was that if a connector has `CREATE SCHEMA` it should also not fail when doing `SHOW CREATE SCHEMA` even when there's no info to show.

Unless I hear otherwise from you I will remove all no-op implementation of these methods and have all those connectors fail when issuing `SHOW CREATE SCHEMA`.

Reply:
BlackHoleMetaData are trivial MemoryMetadata so you could implement that method for them.

I would skip rest for now.

Reply:
Oh missed this. I will add this back for BlockHoleMetadata and MemoryMetadata.